packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/compose/models/HomeViewModel.kt

```diff
@@ -22,9 +26,10 @@ import kotlinx.coroutines.launch
 
 sealed interface HomeAction {
   class OpenApp(val url: String) : HomeAction
-  object RefetchRunningApps : HomeAction
-  object ResetRecentlyOpenedApps : HomeAction
+  data object RefetchRunningApps : HomeAction
+  data object ResetRecentlyOpenedApps : HomeAction
```

Comment:
```suggestion
  object RefetchRunningApps : HomeAction
  object ResetRecentlyOpenedApps : HomeAction
```
I think those don't need to be data classes,